app/extensions/edm/__init__.py

```diff
@@ -88,6 +91,21 @@ class EDMManager(RestManager):
     def __init__(self, pre_initialize=False, *args, **kwargs):
         super(EDMManager, self).__init__(pre_initialize, *args, **kwargs)
 
+    def version_check(self):
+        edm_version = current_app.edm.get_dict('version.dict', None)
+        if edm_version is None or 'date' not in edm_version:
+            log.error('could not determine EDM version')
+            return False
+        if edm_version['date'] >= self.VERSION_REQUIRED:
+            log.info(
```

Comment:
This does seem like it's very verbose and could fill logfiles without giving too much benefit

Reply:
yeah i guess it depends on how often something is calling edm.version_check() ... right now it is only test (and a newly add tasks); but i guess this does beg the question, as you say, what is info and what is debug in general.